### PR TITLE
Add multiply functionality

### PR DESCRIPTION
**Pull request by sdsingh**
The repo was lacking multiplication functionality, which I have gone ahead and added. Warning: there is no form of error checking--just naive multiply.

### CODE REVIEW

File: multiply.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(a, b):
```
Comment (stefanv) [inline multiply.py]: This looks great.  Would you please add a docstring?

**Comment by sdsingh** [comment]
Addressed your comments. Please comment on the new functionality--I think it is a sensible default.

File: multiply.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+def multiply(a, b=None):
+    """Multiply two numbers.
+
+    Parameters
+    ----------
+    a : int or float 
+        First operand.
+    b : int or float (optional)
+        Second operand.
+    Returns
```
Comment (stefanv) [inline multiply.py]: Missing blank line above this one--otherwise looks great.  You even used the numpy docstring format!

File: multiply.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    -------
+    output : int or float
+        Multiplication of parameters, or a*a if b not provided. 
+    """
+    return a * a if b is None else a * b
```
Comment (stefanv) [inline multiply.py]: Mention this functionality in the docstring.

**Comment by sdsingh** [comment]
Updated docstring to fix typo and describe "squaring"

**Pull request merged**